pywr/recorders.py

```diff
@@ -196,7 +202,10 @@ def setup(self):
                 atom = tables.Int32Atom()
             else:
                 atom = tables.Float64Atom()
-            self.h5store.file.create_carray(where, node.name, atom, shape, createparents=True)
+            group_name, node_name = os.path.split(where)
```

Comment:
Is it correct to treat these as filesystem path? Will that cause any problems with `\\` or similar? Why not just `where.split('/', 1)`?